packages/wdio-reporter/package.json

```diff
@@ -22,6 +22,8 @@
     "url": "https://github.com/webdriverio/webdriverio/issues"
   },
   "devDependencies": {
+    "@types/fs-extra": "^9.0.1",
+    "@types/got": "^9.6.11",
```

Comment:
are these needed?

Reply:
The first one is for fs-extra which is used in the index.ts file. It's nice to have types there.
The second one isnt. It is used in the types file that is supposed to be in the repo once your MR is done. I removed it.